### PR TITLE
feat(cxl-lumo-styles): increase default font size for p elements

### PR DESCRIPTION
**Pull request by anoblet**
https://cxlworld.slack.com/archives/C01JABH8AHX/p1720530653218939

### CODE REVIEW

File: packages/cxl-lumo-styles/scss/typography.scss [hunk 1]
```diff
@@ -110,6 +110,13 @@ p {
   .entry-content & {
     font-family: var(--cxl-lumo-font-secondary);
   }
+
+  /**
+   * Improve readability.
+   *
+   * @since 2024.07.09
+   */
```
Comment (pawelkmpt) [inline packages/cxl-lumo-styles/scss/typography.scss:118]: Comment does not provide any value

Comment (pawelkmpt) [reply packages/cxl-lumo-styles/scss/typography.scss]: I mean, just remove it. It's just a font size

**Comment by pawelkmpt** [comment]
I looked at Storybook, compared `master` and this branch. It looks like bigger font size has also impact on layout width in some places. It affects (some) cards too.

Please experiment with such styles on the live site and make sure things won't be broken. Especially landing pages and lessons.

---

BEFORE
1-before

AFTER
1-after

---

BEFORE
2-before

AFTER
2-after

---

BEFORE
3-before

AFTER
3-after

---

BEFORE
4-before

AFTER
4-after

**Comment by anoblet** [comment]
I believe you were correct in suggesting upgrading individual parts would be a better strategy. I updated the PR to increase the font size for .archive-description, .entry-content, .entry-summary.

Instead of merging this quickly, I'll update the live site with one or two rules a day to see if anything breaks. Once we can confirm everything looks good, I'll add it to this PR.

**Comment by pawelkmpt** [comment]
PR for blog is here: https://github.com/conversionxl/aybolit/pull/430

**Pull request closed without merging**